src/intTest/java/com/box/sdk/TestConfig.java

```diff
@@ -0,0 +1,190 @@
+            handler.setLevel(level);
+            logger.addHandler(handler);
+        }
+        return logger;
+    }
+
+    public static String getAccessToken() {
+        if (accessToken == null || accessToken.equals("")) {
+            accessToken = getProperty("accessToken");
+        }
+
+        return accessToken;
+    }
+
+    public static void setAccessToken(String accessToken) {
+        TestConfig.accessToken = accessToken;
+    }
+
+    public static String getRefreshToken() {
+        if (refreshToken == null || refreshToken.equals("")) {
+            refreshToken = getProperty("refreshToken");
+        }
+
+        return refreshToken;
+    }
+
+    public static void setRefreshToken(String refreshToken) {
+        TestConfig.refreshToken = refreshToken;
+    }
+
+    public static String getClientID() {
+        if (clientID == null || clientID.equals("")) {
+            clientID = getProperty("clientID");
+        }
+
+        return clientID;
+    }
+
+    public static String getClientSecret() {
+        if (clientSecret == null || clientSecret.equals("")) {
+            clientSecret = getProperty("clientSecret");
+        }
+
+        return clientSecret;
+    }
+
+    public static String getCollaborator() {
+        if (collaborator == null || collaborator.equals("")) {
+            collaborator = getProperty("collaborator");
+        }
+
+        return collaborator;
+    }
+
+    public static String getCollaboratorID() {
+        if (collaboratorID == null || collaboratorID.equals("")) {
+            collaboratorID = getProperty("collaboratorID");
+        }
+
+        return collaboratorID;
+    }
+
+    public static String getEnterpriseID() {
+        if (enterpriseID == null || enterpriseID.equals("")) {
+            enterpriseID = getProperty("enterpriseID");
+        }
+
+        return enterpriseID;
+    }
+
+    public static String getPrivateKey() {
+        if (privateKey == null || privateKey.equals("")) {
+            privateKey = getProperty("privateKey");
+        }
+
+        return privateKey;
+    }
+
+    public static String getPrivateKeyPassword() {
+        if (privateKeyPassword == null || privateKeyPassword.equals("")) {
+            privateKeyPassword = getProperty("privateKeyPassword");
+        }
+
+        return privateKeyPassword;
+    }
+
+    public static String getPublicKeyID() {
+        if (publicKeyID == null || publicKeyID.equals("")) {
+            publicKeyID = getProperty("publicKeyID");
+        }
+
+        return publicKeyID;
+    }
+
+    public static String getTransactionalAccessToken() {
+        if (transactionalAccessToken == null || transactionalAccessToken.equals("")) {
+            transactionalAccessToken = getProperty("transactionalAccessToken");
+        }
+
+        return transactionalAccessToken;
+    }
+
+    private static String getProperty(String name) {
+        Properties configProperties = loadProperties();
```

Comment:
It is lazy loading. If you check loadProperties it sets a property field. Now when calling loadProperties it will returned already loaded data if this field is set, if not it will read from a file.